x-pack/libbeat/management/managerV2.go

```diff
@@ -169,14 +168,12 @@ func NewV2AgentManager(config *conf.C, registry *reload.Registry) (lbmanagement.
 		agentClient = client.NewV2(c.InsecureGRPCURLForTesting,
 			"", // Insecure connection for test, no token needed
 			client.VersionInfo{
-				Name:    "beat-v2-client-for-testing",
-				Version: version.GetDefaultVersion(),
+				Name: "beat-v2-client-for-testing",
 			}, client.WithGRPCDialOptions(grpc.WithTransportCredentials(insecure.NewCredentials())))
 	} else {
 		// Normal Elastic-Agent-Client initialisation
 		agentClient, _, err = client.NewV2FromReader(os.Stdin, client.VersionInfo{
-			Name:    "beat-v2-client",
-			Version: version.GetDefaultVersion(),
+			Name: "beat-v2-client",
```

Comment:
We aren't setting the `BuildHash` in `client.VersionInfo`.

```
// VersionInfo is the version information for the connecting client.
type VersionInfo struct {
	// Name is the name of the program.
	Name string
	// Meta is any extra metadata information about the version.
	Meta map[string]string
	// BuildHash is the VCS commit hash the program was built from.
	BuildHash string
}
```

We are only setting the `"commit"` property in the meta information. We aren't setting this in the `NewV2` case above either.

Reply:
true, then we should not need the `commit` in the metadata. I'll remove it ok?

Reply:
Agreed we don't need the commit in the metadata anymore